api/rest/v1/bulk_properties.md

```diff
@@ -0,0 +1,339 @@
+---
+layout: default
+title: Properties
+parent: v1 REST
+grand_parent: API
+nav_exclude: true
+published: false
+permalink: /api/rest/v1/bulk/properties
+---
+
+# /v1/bulk/properties
+
+Get all [properties](/glossary.html#property) associated with a specific entity, for multiple entities.
```

Comment:
Please change "entity" to "node" throughout

Reply:
Done! Also updated entity->node and entities->nodes in the request and response signature. Please double check I got the format right.